feature-libs/user/profile/styles/_new-profile.scss

```diff
@@ -0,0 +1,57 @@
+%cx-new-profile {
+
+  .myaccount-enhancedUI-headertext {
+    font-size: $h3-font-size;
+    font-weight: var(--cx-font-weight-semi);
+    color: $text;
+    padding-bottom: 1.5rem;
+  }
+
+  .myaccount-enhancedUI-editButton {
+    font-size: $font-size-base;
+    color: $primary;
+    border: none;
+    background-color: transparent;
+  }
+
+  .myaccount-enhancedUI-text-head {
+    font-weight: var(--cx-font-weight-semi);
+    min-width: 7.5rem;
+    color: $background-dark;
```

Comment:
* Please use our `—cx-color` variables for colours and `@include type()` mixin for predefined fonts. We are no longer using Bootstrap variables.

Reply:
Updated, Pls check